index.js

```diff
@@ -96,8 +118,58 @@ const ScrollableTabView = React.createClass({
     }
   },
 
+  updateSceneKeys(children,sceneKeys=[],currentPage){
+    let newKeys=[]
+    children.forEach((child,idx) => {
+      let key=this._makeSceneKey(child,idx)
+      if(this._keyExists(sceneKeys,key) || currentPage == idx) newKeys.push(key)
+    })
+    return newKeys;
+  },
+
+  _keyExists(sceneKeys,key){
+    return sceneKeys.findIndex((sceneKey)=> key == sceneKey) > -1
+  },
+
+  _makeSceneKey(child,idx){
+    return child.props.tabLabel + '_' + idx
+  },
+
   renderScrollableContent() {
     if (Platform.OS === 'ios') {
+      let scenes=[]
```

Comment:
I think this code same for both platforms. Please create a function or component instead of duplicating code.